src/frontend/src/components/VerticalDetailDisplay.tsx

```diff
@@ -0,0 +1,36 @@
+/*
+ * This file is part of FinishLine by NER and licensed under GNU AGPLv3.
+ * See the LICENSE file in the repository root folder for details.
+ */
+
+import { useTheme } from '@mui/material';
+import Typography from '@mui/material/Typography';
+import { Box } from '@mui/system';
+
+interface VerticalDetailDisplayProps {
+  label: string;
+  content: string;
+  paddingRight?: number;
```

Comment:
you could even make this an sx prop if you want, and let people override the styles when needed, up to u tho